transmit-lib/src/cmr/transmit/kms.clj

```diff
@@ -51,7 +84,7 @@
    :measurement-name "measurementname?format=csv"
    :concepts "idnnode?format=csv"
    :iso-topic-categories "isotopiccategory?format=csv"
-   :related-urls "rucontenttype?format=csv"
+   :related-urls (env-varient "rucontenttype?format=csv" if-not-production "&version=DRAFT")
```

Comment:
Seems that the code doesn't need to be this fancy.  the logic is so simple
(if production?
  url
  (str url varient)) 
We don't need to pass if-not-production as a function, and have an additional env-variant function.